modules/web/src/app/cluster/details/cluster/mla/alertmanager-config/component.ts

```diff
@@ -67,10 +68,14 @@ export class AlertmanagerConfigComponent implements OnInit, OnDestroy {
     private readonly _settingsService: SettingsService,
     private readonly _datacenterService: DatacenterService,
     private readonly _addonService: AddonService,
+    private readonly _projectService: ProjectService,
     @Inject(DOCUMENT) private readonly _document: Document
   ) {}
 
   ngOnInit(): void {
+    this._projectService.selectedProject
+      .pipe(take(1))
+      .subscribe(project => (this._grafanaOrgId = project.annotations['mla.k8c.io/organization']));
```

Comment:
It would be great if you can add this static value in an enum. For example, 
```
enum ProjectAnnotation {
  GrafanaOrgID = 'mla.k8c.io/organization',
}
```
And then put it in `shared/entity/project.ts`

Similar to how we have `ClusterAnnotation` in `shared/entity/cluster.ts`